\documentclass[11pt,a4paper,sans]{moderncv}

% Adjust margins and layout
\usepackage[a4paper, margin=1in, bottom=1in]{geometry}

% ModernCV themes
\moderncvstyle{banking}
\moderncvcolor{blue}
\nopagenumbers{} % Suppress page numbers

% Required packages for ModernCV
\usepackage{fontspec}
\usepackage{fontawesome5}
\usepackage{academicons}
\usepackage{multicol}

% Fancyhdr for custom footer
\usepackage{fancyhdr}
\pagestyle{fancy}
\fancyhf{}
\fancyfoot[L]{\normalsize References available only upon request and if shortlisted} % Larger footer text
\setlength{\footskip}{50pt} % Push footer closer to the page's bottom edge
\renewcommand{\headrulewidth}{0pt} % Remove header line
\renewcommand{\footrulewidth}{0pt} % Remove footer line

% Adjust list spacing globally
\usepackage{enumitem}
\setlist[itemize]{topsep=0pt, itemsep=1pt, leftmargin=1.5em}

% Set fonts to Garamond
\setmainfont{EB Garamond}
\setsansfont{EB Garamond}
\setmonofont{Courier New}

% Personal data
\name{Gerald Amiel}{Ballena}
\title{Bioinformatics Engineer}
\email{gmballena@up.edu.ph}
\social[github]{GABallena}
\social[linkedin]{gerald-amiel-ballena}
\extrainfo{\aiOrcid \ \href{https://orcid.org/0009-0000-8857-9755}{orcid.org/0009-0000-8857-9755}}

\renewcommand{\makecvtitle}{
	\vspace*{-2em}
	\begin{center}
		{\Huge \textbf{Gerald Amiel Ballena}} \\[0.5em]
		\faEnvelope \ \href{mailto:gmballena@up.edu.ph}{gmballena@up.edu.ph} \quad
		\faGithub \ \href{https://github.com/GABallena}{github.com/GABallena} \quad
		\faLinkedin \ \href{https://linkedin.com/in/gerald-amiel-ballena}{linkedin.com/in/gerald-amiel-ballena} \\[0.5em]
		\faMapMarker \ Metro Manila, Philippines \quad
		\aiOrcid \ \href{https://orcid.org/0009-0000-8857-9755}{orcid.org/0009-0000-8857-9755}
	\end{center}
	\vspace{1.5em}
}

\begin{document}
	
	\makecvtitle
	
	\section{Professional Summary}
	Versatile Bioinformatics Engineer with extensive experience in computational biology, bioinformatics pipelines, and data analysis. Highly skilled in automating workflows using Snakemake and Python, optimizing tools for high-throughput sequence analysis, and ensuring reproducibility and efficiency. Adept at collaborating with research teams to produce clear, reliable solutions while solving complex bioinformatics challenges. Proficient in bioinformatics tools and skilled in data visualization and pipeline development.
	
	\section{Experience}
	\textbf{2024 â€“ Present} \hfill \textbf{Project Technical Specialist} \\
	\textit{University of the Philippines, College of Public Health} \\[-1em]
	\begin{itemize}
		\item Developed bioinformatics workflows and automated pipelines for large-scale sequencing data analysis.
		\item Collaborated with teams across public health, microbiology, and environmental science to produce clear, reproducible research data.
		\item Wrote Python scripts to automate data processing, error-checking, and visualizations, streamlining data management and reducing errors by 90\%.
		\item Enhanced the analysis pipeline, leading to actionable insights for public health surveillance and research.
	\end{itemize}
	
	\section{Education}
	\subsection{University of the Philippines Diliman}
	\cvitem{Graduated}{July 2022}
	\cvitem{Thesis}{In silico assessment of the association of pathogenicity and metal-resistance potential of \textit{Fusarium} spp.\dotfill \href{https://www.biorxiv.org/content/10.1101/2022.10.12.511937v1}{Pre-print Link}}
	\cvitem{Accomplishments}{DOST ASTHRDP-Scholarship \\ Calculated Effective GPA (MS): 1.72}
	
	\subsection{University of the Philippines Baguio}
	\cvitem{Graduated}{June 2018} 
	\cvitem{Thesis}{\textbf{Bioelectrocatalysis by Novel Electrogenic Alkaliphilic Bacteria \textit{Bacillus} sp. BAB-3442 Using Dual-Chambered Microbial Fuel Cell} \dotfill Poster Presentation (\texttt{PSM 47})}
	
	\section{Skills}
	\subsection{Technical Skills}
	\cvitem{Programming Languages}{Python (Fluent), R (Advanced), Bash (Intermediate), Perl \& BioPerl (Intermediate), C++ (Fair).}
	\cvitem{Bioinformatics Tools}{Kraken2, MetaPhlAn, MEGAHIT, Snakemake, Prokka, Jellyfish, BUSCO, and others.}
	\cvitem{Workflow Automation}{Snakemake, Conda, YAML (Configuration Files).}
	\cvitem{Data Visualization}{ggplot2, Plotly, matplotlib, QGIS.}
	\cvitem{High-Performance Computing}{Slurm, Docker.}
	
	\subsection{Soft Skills}
	\cvitem{Collaboration}{Proficient in working with research teams, ensuring smooth project execution and data delivery.}
	\cvitem{Problem-Solving}{Skilled at troubleshooting issues within bioinformatics pipelines and proposing effective solutions.}
	\cvitem{Technical Writing}{Expert in preparing clear technical reports and scientific documentation, focusing on precision and clarity.}
	
	\section{Certifications and Relevant Coursework}
	\subsection{Certifications}
	\cvitem{Fundamentals}{AI Fundamentals \dotfill 2024}
	\cvitem{Fundamentals}{Data Literacy \dotfill 2024}
	\cvitem{Advanced}{PH527x: Principles, Statistical and Computational Tools for Reproducible Data Science \dotfill 2024}
	
	\subsection{Relevant Courses}
	\cvitem{Introductory}{Introduction to Data Engineering \dotfill 2020}
	\cvitem{Introductory}{COVID-19 Contact Tracing \dotfill 2020}
	\cvitem{Intermediate}{Biostatistics in Public Health \dotfill 2020}
	\cvitem{Intermediate}{Genomic Data Science \dotfill 2024}
	\cvitem{Intermediate}{Genomic Analysis track \dotfill 2024}
	
	\cvitem{Advanced}{Hyperparameter Tuning in R, Designing Machine Learning Workflows in Python \dotfill 2024}
	\cvitem{Advanced}{Bioinformatics via Coursera \dotfill 2024}
	(\textit{University of California, San Diego}) 
	\begin{itemize}
		\item Finding Hidden Messages (with Honors)
	\end{itemize}
	
	\subsection{Professional Development}
	\cvitem{Ongoing}{Ensemble Methods in Python (Bagging, Boosting, Stacking) \ Introduction to AWS \ Supervised Learning with scikit-learn}
	
	\section{Scripts and Workflows}
	
	\cvitem{Key Pipelines and Workflows}{
		Developed and implemented scalable workflows and pipelines using Snakemake, Python, and Bash for metagenomic analysis, diversity profiling, and bioinformatics tool management. Highlights include:
		\begin{itemize}
			\item \textbf{Metagenomic Analysis Pipeline:} Automated workflows for trimming, taxonomic profiling (\textbf{Kraken2}, \textbf{Bracken}), and diversity calculations (\textbf{scikit-bio}), ensuring high reproducibility and scalability.
			\item \textbf{Comprehensive Binning Workflow:} Designed workflows for assembly, binning (\textbf{MetaWRAP}, \textbf{MEGAHIT}), and MAG validation (\textbf{CheckM2}), streamlining large-scale metagenomic projects.
			\item \textbf{Plasmid and ARG Analysis:} Built pipelines for plasmid detection and antimicrobial resistance profiling using \textbf{metaSPAdes}, \textbf{PlasmidFinder}, and \textbf{RGI}.
			\item \textbf{Diversity Analysis:} Developed Python-based scripts and R workflows for alpha/beta diversity metrics (Shannon, Chao1, Bray-Curtis) and visualizations using \textbf{ggplot2}.
		\end{itemize}
	}
	
\end{document}
